### PR TITLE
fence_compute: Create nova client with API 2.11

### PR DESCRIPTION
**Pull request by vuntz**
The force_down API requires to use version 2.11 or later.

### CODE REVIEW

**Comment by beekhof** [comment]
Is there a way we can have this fall back to plain '2' if 2.11 is not available?
(Safe since we drop back to the wait loop if the force_down call fails)

**Comment by beekhof** [comment]
novaclient.exceptions.UnsupportedVersion: Invalid client version '2.11'. must be one of: 3, 2, 1.1

**Comment by vuntz** [comment]
> novaclient.exceptions.UnsupportedVersion: Invalid client version '2.11'. must be one of: 3, 2, 1.1

Hrm, so you use an old version of novaclient, I guess. I added a commit that should make it work.

**Comment by aspiers** [comment]
@vuntz CI needs fixing somehow.

**Comment by oalbrigt** [comment]
The issue has been fixed in master. Can you rebase and patch the updated master branch?

**Comment by vuntz** [comment]
> The issue has been fixed in master. Can you rebase and patch the updated master branch?

Thanks, done.

File: fence/agents/compute/fence_compute.py [hunk 1]
```diff
@@ -347,8 +352,12 @@ def main():
 		elif options["--action"] in ["monitor", "status"]:
 			sys.exit(0)
 
-	# The first argument is the Nova client version
-	nova = nova_client.Client('2',
+	if LooseVersion(novaclient.__version__) <= LooseVersion('2.26.0') :
```
Comment (beekhof) [inline fence/agents/compute/fence_compute.py]: Is that going to do anything useful if LooseVersion couldn't be imported?
Surely the agent would just throw an error and abort

Comment (vuntz) [inline fence/agents/compute/fence_compute.py]: See the `fail_usage` call earlier if it cannot be imported...

**Comment by vuntz** [comment]
I think this patch is obsoleted by aa8b29615274feb0e1904b9a887a11ac89d95d24. I don't get why my patch wasn't merged months ago, though...

**Pull request closed without merging**